wallpaper-bindings/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "wallpaper-bindings"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+more-wallpapers = {version = "0.3.1", features = ["fallback"]}
+tauri = { version = "1.5", features = ["shell-open"] }
+
+[lib]
+name = "jdtwallpaper"
+path = "src/lib.rs"
+
+[[bin]]
```

Comment:
Added an executable as well just so that it was easy to test some of the wallpaper functionality directly